src/pages/CreateInstitutionPage.jsx

```diff
@@ -287,6 +287,7 @@ function CreateInstitutionPage(props) {
                                         id="name"
                                         className="form-control bg-light-pastel-blue color-grey fw-medium fs-5 border-0 rounded-4 mb-3"
                                         onChange={(e) => setInstitution({ ...institution, name: e.target.value })}
+                                        required
                                     />
```

Comment:
```suggestion
                                        onChange={(e) => setInstitution({ ...institution, name: e.target.value })}
                                        minLength="1"
                                        maxLength="255"
                                        required
                                    />
```
- Nome também possui limites inferior e superior na API (entendo que mínimo de um é redundante na sugestão);